perception/detected_object_validation/src/object_lanelet_filter.cpp

```diff
@@ -102,8 +161,15 @@ void ObjectLaneletFilterNode::objectCallback(
       (label == Label::BICYCLE && filter_target_.BICYCLE) ||
       (label == Label::PEDESTRIAN && filter_target_.PEDESTRIAN)) {
       Polygon2d polygon;
+
+      // set tf_target2objects
+      transform_point.setTransformTarget2Objects(
+        input_msg->objects.at(index).kinematics.pose_with_covariance.pose);
+
       for (const auto & point : footprint.points) {
-        polygon.outer().emplace_back(point.x + position.x, point.y + position.y);
```

Comment:
I think the problem is caused this code.
This code don't care the quaternion.

Reply:
Yes. So I use tf to transform point with translation and rotational transfer.

Reply:
@yukkysaito 
Use transformPoint and  care the quaternion in https://github.com/autowarefoundation/autoware.universe/pull/1981/commits/35fa236fb5c5c2e99368165d4adf01fabfa16bb7